src/data/finset/pointwise.lean

```diff
@@ -959,6 +971,17 @@ end
 
 open_locale pointwise
 
+@[to_additive] lemma smul_image [decidable_eq β] [decidable_eq γ] [has_smul α β] [has_smul α γ]
+  (f : β → γ) (s : finset β) (a : α) :
+  (∀ b, a • f b = f (a • b)) → a • s.image f = (a • s).image f :=
+image_comm
```

Comment:
This feels more natural to me as
```suggestion
@[to_additive] lemma smul_image [decidable_eq β] [decidable_eq γ] [has_smul α β] [has_smul α γ]
  (f : β → γ) (s : finset β) (a : α) :
  (∀ b, f (a • b) = a • f b) → (a • s).image f = a • s.image f :=
image_comm
```
since then the hypothesis is of the form `map_smul`